tools/jenkins/common.groovy

```diff
@@ -0,0 +1,44 @@
+def prepare(build_info) {
+  stage("Prepare"){
+    if(!env.BRANCH_NAME.contains("PR-")) {
+      slackSend channel: "#jenkins", message: "Build Started: <${env.JOB_URL}|${env.JOB_NAME}> <${env.BUILD_URL}|#${env.BUILD_NUMBER}> <${env.CHANGE_URL ?: 'https://github.com/ConduceInc/everything'}|(GitHub)>"
+    }
+
+    sh "rm -rf venv"
+    sh "sudo rm -rf htmlcov"
+    sh "sudo rm -rf doc/build"
+    sh "virtualenv --python=/usr/bin/python3.4 venv"
+    sh ". venv/bin/activate && pip install -r requirements-dev.txt"
+
+    milestone()
+  }
+}
+
+
+def test_and_build() {
+  withEnv(["GLSL_PATH=${env.PWD}/shared/python/victor/glsl/"]) {
```

Comment:
this repo doesn't have a `shared/python/victor/glsl` directory.  Leftover copy/paste?